op-e2e/actions/l1_miner.go

```diff
@@ -29,11 +29,11 @@ type L1Miner struct {
 }
 
 // NewL1Miner creates a new L1Replica that can also build blocks.
-func NewL1Miner(log log.Logger, genesis *core.Genesis) *L1Miner {
-	rep := NewL1Replica(log, genesis)
+func NewL1Miner(log log.Logger, genesis *core.Genesis) (*L1Miner, func() error) {
+	rep, done := NewL1Replica(log, genesis)
 	return &L1Miner{
 		L1Replica: *rep,
-	}
+	}, done
```

Comment:
Maybe we can use the `testing.Cleanup()` functionality to close the geth node, instead of passing around a close function?

Reply:
This is neat, didn't know that was a feature. Will look into that.